stan/math/prim/arr/err/is_nonzero_size.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef STAN_MATH_PRIM_ARR_ERR_IS_NONZERO_SIZE_HPP
+#define STAN_MATH_PRIM_ARR_ERR_IS_NONZERO_SIZE_HPP
+
+namespace stan {
+namespace math {
+
+/**
+ * Returns <code>true</code> if the specified matrix/vector is size nonzero.
+ *
+ * @tparam T_y Type of container, requires class method <code>.size()</code>
+ *
```

Comment:
Please remove the spaces between these comment docs.  Our autoformatting doesn't catch this, but we want to be conservative with vertical space.  That goes for here and elsewhere.  Thanks.